### DISCUSSION TITLE
[Homebrew] command not found after installing nerdctl

### DISCUSSION THREAD

**Discussion by vmwarelab** [containerd/nerdctl#3737] [Q&A]
Hi 

i have an ubuntu 22.04 and installed brew on it .. then proceeded with brew install nerdctl 

when executing sudo nerdctl argument I get command not found

**Comment by apostasie** [comment]
@vmwarelab:

Do you have brew binary install location in your PATH?

**Reply by apostasie** [reply]
Also check what happens without `sudo`.
You likely have a PATH problem here.

**Reply by vmwarelab** [reply]
I have yet to check the path. I will check. I thought the installation would have automatically added it to the path
. What path do I need to add? It doesn't say anything in the installation instructions

**Reply by apostasie** [reply]
The PATH "problem" is about `brew` usage - nothing `nerdctl` can do about.
Have you used brew on linux before?
Were you successful with other `brew install` commands?
Also, sudo comes with its own issues.
